gds-sso.gemspec

```diff
@@ -14,7 +14,7 @@ Gem::Specification.new do |s|
   s.description = "Client for GDS' OAuth 2-based SSO"
   s.license     = "MIT"
 
-  s.required_ruby_version = ">= 2.6"
+  s.required_ruby_version = ">= 2.7" # rubocop:disable Gemspec/RequiredRubyVersion
```

Comment:
Can you explain why the rubocop rule is needed?

Reply:
I added this because when I increased the required version (based on the fact that we're no longer testing on 2.6) Rubocop complained that there was a mismatch with the `TargetRubyVersion`, but it wasn't clear where `TargetRubyVersion` was being set (`.rubocop.yml` doesn't have one) so I interpreted that as being a false positive.

On reflection it might be just an incorrect default and so setting it in `.rubocop.yml` might work?